### PR TITLE
Images issue165

### LINKED ISSUE #165: Image replacements
Multiple images within the COP need replacing, such as figure 3.d. These will all be replaced as a single issue.

### PR DESCRIPTION
**Pull request by tashalouizamc**
Replaced images 3a to 3e, including updating figure labels and alt-text where necessary. Deleted defunct images for audit logo and tech notes.

### CODE REVIEW

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
You've removed the 91-counter-audit-logo.jpg and d-tech-notes-cover.jpg, but not the corresponding references?

**Comment by tashalouizamc** [comment]
> You've removed the 91-counter-audit-logo.jpg and d-tech-notes-cover.jpg, but not the corresponding references?

As far as I know, the audit logo is only referenced in the R5 version of section 9, which is replaced by pr #185 and the tech notes cover is only in section d, which is being deleted as part of the clean up issue

**Comment by tashalouizamc** [comment]
Merging to facilitate clean up and text edit issues

File: source/03-specifications/01-counter-reports-for-consumers.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ Table 3.a (below): COUNTER Reports
 
 Figure 3.a (below) provides an example of how the user interface could look. The user will be presented with an interface that allows them to select usage dates, one or more Metric_Types, Data_Types, Access_Types, etc. and indicate if the filter columns are to be included. Including the column will cause usage to be broken out by individual values for the selected filter, whereas not including the column will result in usage being summarized for the selected filter.
 
-.. figure:: ../_static/img/3a-reporting-ui-sample.png
+.. figure:: ../_static/img/Figure-3a.png
```
Comment (berndoberknapp) [inline source/03-specifications/01-counter-reports-for-consumers.rst:73]: The option to include the "called for" columns Access_Type, Access_Method and YOP in the report is missing in the new image (Data_Type is now required, and the checkbox for Metric_Type was wrong in the old image).

File: source/03-specifications/02-formats-for-counter-reports.rst [hunk 2]
```diff
@@ -9,21 +9,21 @@ Formats for COUNTER Reports
...
 
-All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the “Journal Requests (Controlled)” Standard View. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
+All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the Title Report. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
 
-.. figure:: ../_static/img/3b-standard-view-sample.png
-   :alt: Standard View sample
+.. figure:: ../_static/img/Figure-3b.png
```
Comment (berndoberknapp) [inline source/03-specifications/02-formats-for-counter-reports.rst:14]: The Title Report needs to be scaled down a lot to fit on the page and is therefore difficult to read. I would suggest to only use the left half of the current images (with the fixes below).

The Release must be formatted as string. The Metric_Types filter is the default, therefore the list must be omitted. Attributes_To_Show is missing.

File: source/03-specifications/02-formats-for-counter-reports.rst [hunk 3]
```diff
@@ -9,21 +9,21 @@ Formats for COUNTER Reports
...
+   :alt: Title Report sample
    :align: center
    :width: 80%
 
-.. centered:: Figure 3.b: Sample “Journal Requests (Controlled)” Standard View
+.. centered:: Figure 3.b: Sample Title Report
 
-.. figure:: ../_static/img/3c-tabular-report-layout.png
-   :alt: Tabular Report layout
+.. figure:: ../_static/img/Figure-3c.png
```
Comment (berndoberknapp) [inline source/03-specifications/02-formats-for-counter-reports.rst:21]: Please replace "Report / Item" (I have no idea why that term was used in the old image) with "Report Attributes". The singular/plural should be consistent (e.g. Metric types but Reporting period total). I would prefer the same upper/lower case as in the text, e.g. Metric Types.

Comment (berndoberknapp) [reply source/03-specifications/02-formats-for-counter-reports.rst:21]: I forgot to mention that I would prefer vertical dotted line between the areas as in the old image.

File: source/03-specifications/03-counter-report-common-attributes-and-elements.rst [hunk 4]
```diff
@@ -371,12 +371,12 @@ Table 3.r (below): List of Metric_Types for Searches
 
 This group of Metric_Types represents activities where content items were retrieved (Requests) or information about a content item (e.g. an abstract) was examined (Investigations). Any user activity that can be attributed to a content item will be considered an Investigation including downloading or viewing the item. Requests are limited to user activity related to retrieving or viewing the content item itself. The figure below provides a graphical representation of the relationship between Investigations and Requests.
 
-.. figure:: ../_static/img/3e-investigations-and-requests.png
+.. figure:: ../_static/img/Figure-3e.png
```
Comment (berndoberknapp) [inline source/03-specifications/03-counter-report-common-attributes-and-elements.rst:374]: I find the Investigations part in the old image easier to understand, especially:

- The terms are ambiguous without the "user action" part. Only a user action that results for example in a metdata record to be shown is an Investigation, but not a metadata record for example shown in a result list.
- The outgoing link actions are missing. In my opinion these are important examples that are not obvious.

**Review by berndoberknapp — COMMENTED** [review]
Please see the inline comments.

**Pull request merged**